fontbe/src/os2.rs

```diff
@@ -80,6 +263,51 @@ fn apply_metrics(os2: &mut Os2, metrics: &GlobalMetricsInstance) {
     os2.us_win_descent = metrics.os2_win_descent.ot_round();
 }
 
+fn add_unicode_range_bits(add_to: &mut HashSet<usize>, codepoint: usize) {
+    // Is there any range this codepoint _might_ fit?
+    let maybe_idx = UNICODE_RANGES
+        .binary_search_by(|(from, to, _)| match codepoint {
+            _ if codepoint < *from => Ordering::Greater,
+            _ if codepoint > *to => Ordering::Less,
+            _ => Ordering::Equal,
+        })
+        .ok();
+    if let Some(idx) = maybe_idx {
+        add_to.insert(UNICODE_RANGES[idx].2);
+    }
```

Comment:
code golf:

```suggestion
    // Is there any range this codepoint _might_ fit?
    if let Ok(idx) = UNICODE_RANGES
        .binary_search_by(|(from, to, _)| match (from.cmp(&codepoint), to.cmp(&codepoint)) {
        (Ordering::Greater, _) => Ordering::Greater,
        (_, Ordering::Less) => Ordering::Less,
        _ => Ordering::Equal,
        }) {
            add_to.insert(UNICODE_RANGES[idx].2);
        }      
    }
```

Reply:
I tried that but I find it harder to read than doing it in separate steps and afaik there is no downside?

Reply:
...the comment is out of date!

Reply:
I tried it again adn I still think it's harder to read so I'm inclined to leave this as-is.